packages/grid/x-data-grid-pro/src/tests/editComponents.DataGridPro.test.tsx

```diff
@@ -623,7 +623,7 @@ describe('<DataGridPro /> - Edit components', () => {
       userEvent.mousePress(screen.queryAllByRole('option')[1]);
       clock.runToLast();
       expect(screen.queryByRole('listbox')).to.equal(null);
-      fireEvent.keyDown(screen.getByRole('button', { name: 'Adidas' }), { key: 'Enter' });
+      fireEvent.keyDown(screen.getByRole('combobox'), { key: 'Enter' });
```

Comment:
@mui/xgrid this solves https://app.circleci.com/pipelines/github/mui/mui-x/45749/workflows/9c314f13-57ea-433a-8816-783e8ff7510f/jobs/262589
`Select` is no longer a `button`, but a `combobox` and `name` no longer work in such cases.
The component (Select) might be missing some information to expose it, but in any case, there is only one `combobox`, thus, omitting the explicit `name` in this case seems like a decent compromise. 🤔